src/lib/utils/auto-prefixer.ts

```diff
@@ -10,24 +10,21 @@
 export function applyCssPrefixes(target) {
   for (let key in target) {
 
-    let value = target[key];
+    let value = target[key] || "";
+    let boxValue = toBoxValue(value);
+    let boxDirection = toBoxDirection(value);
+    let boxOrient = toBoxOrient(value);
 
     switch (key) {
       case 'display':
         if (value === 'flex') {
           target['display'] = [
             '-webkit-box',
-            '-moz-box',
-            '-ms-flexbox',
-            '-webkit-flex',
```

Comment:
I think you removed the wrong one here; you want to keep `-webkit-flex` and remove `-webkit-box`
(same for the other related properties throughout)